presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveTpchParquetQueries.java

```diff
@@ -0,0 +1,152 @@
+import com.facebook.presto.tests.DistributedQueryRunner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static org.testng.Assert.assertNotNull;
+
+public abstract class TestHiveTpchParquetQueries
+        extends TestHiveTpchQueries
+{
+    protected TestHiveTpchParquetQueries(boolean useThrift)
+    {
+        super(useThrift);
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        String prestoServerPath = System.getProperty("PRESTO_SERVER");
+        String dataDirectory = System.getProperty("DATA_DIR");
+        String workerCount = System.getProperty("WORKER_COUNT");
+
+        assertNotNull(prestoServerPath);
+        assertNotNull(dataDirectory);
+
+        return HiveExternalWorkerQueryRunner.createNativeQueryRunner(
+                dataDirectory,
+                prestoServerPath,
+                Optional.ofNullable(workerCount).map(Integer::parseInt),
+                0,
+                useThrift,
+                ImmutableMap.of(
+                        "optimizer.optimize-hash-generation", "false",
+                        "parse-decimal-literals-as-double", "true",
+                        "inline-sql-functions", "false",
+                        "use-alternative-function-signatures", "true",
+                        "experimental.table-writer-merge-operator-enabled", "false"),
+                ImmutableMap.of(
+                        "hive.storage-format", "PARQUET",
+                        "hive.pushdown-filter-enabled", "true",
+                        "hive.parquet.pushdown-filter-enabled", "true"));
+    }
+
+    @Override
+    protected ExpectedQueryRunner createExpectedQueryRunner()
+            throws Exception
+    {
+        String dataDirectory = System.getProperty("DATA_DIR");
+        String security = "sql-standard";
+
+        DistributedQueryRunner queryRunner = HiveQueryRunner.createQueryRunner(
+                ImmutableList.of(),
+                ImmutableMap.of(
+                        "parse-decimal-literals-as-double", "true",
+                        "regex-library", "RE2J",
+                        "offset-clause-enabled", "true"),
+                security,
+                ImmutableMap.of(
+                        "hive.storage-format", "PARQUET",
+                        "hive.pushdown-filter-enabled", "true"),
+                Optional.of(Paths.get(dataDirectory)));
+
+        createLineitemParquet(queryRunner);
+        createOrdersParquet(queryRunner);
+        createNationParquet(queryRunner);
+        createCustomerParquet(queryRunner);
+        createPartParquet(queryRunner);
+        createPartSuppParquet(queryRunner);
+        createRegionParquet(queryRunner);
+        createSupplierParquet(queryRunner);
+
+        return queryRunner;
+    }
+
+    private static void createLineitemParquet(QueryRunner queryRunner)
```

Comment:
If this class is already defined for parquet only, then there is no need in each method to still have Parquet in the name.